eth_p2p/rlpx.nim

```diff
@@ -1264,9 +1304,18 @@ proc rlpxAccept*(node: EthereumNode,
     initSecretState(handshake, authMsg, ^ackMsg, result)
 
     let listenPort = transport.localAddress().port
-    await result.hello(devp2pVersion, node.clientId,
-                       node.rlpxCapabilities, listenPort.uint,
-                       node.keys.pubkey.getRaw())
+
+    when defined(useSnappy):
+      let peerProtocolVersion = if result.snappyEnabled: devp2pSnappyVersion.uint
```

Comment:
Let's define a helper template `peer.baseProtocolVersion` that expands to `devp2pVersion` when snappy is disabled and to the branched logic above when snappy is enabled. Then, a lot of the `when` statements in the module would become unnecessary.

Reply:
Also, please align multi-line ifs like this:

```nim
var foo = if x: bar
          else: baz
```